resources/views/settings/labels.blade.php

```diff
@@ -122,7 +122,30 @@ class="table table-striped snipe-table"
                                 </div>
                             </div>
 
-                            <!-- Use Asset Logo -->
+                            <!-- fonts, variable -->
+                            <div class="form-group">
+                                <div class="col-md-3 text-right">
```

Comment:
Do we need text-right here?

Reply:
I think so - everything else is text-right so that the left column - the labels - is next to the right column - the form controls - this just matches that. Happy to change it though if you think that's what's needed?